src/components/common/icons.tsx

```diff
@@ -40,6 +40,21 @@ export const FirestoreIcon = svgIcon(
   </svg>
 );
 
+export const AuthIcon = svgIcon(
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <path
+      d="M10.8 7.79999C10.8 9.45684 9.45681 10.8 7.79995 10.8C6.1431 10.8 4.79995 9.45684 4.79995 7.79999C4.79995
+      6.14313 6.1431 4.79999 7.79995 4.79999C9.45681 4.79999 10.8 6.14313 10.8 7.79999ZM1.19995
+      16.68V19.2H14.4V16.68C14.4 14.2745 11.445 13.2 7.79995 13.2C4.15487 13.2 1.19995
+      14.2745 1.19995 16.68ZM16.2 10.8C17.8568 10.8 19.2 9.45684 19.2 7.79999C19.2 6.14313 17.8568 4.79999 16.2
+      4.79999C14.5431 4.79999 13.2 6.14313 13.2 7.79999C13.2 9.45684 14.5431 10.8 16.2 10.8ZM22.8 19.2H15.6V16.68C15.6
+      15.25 14.9331 14.1526 13.7752 13.3811C14.5258 13.2593 15.3439 13.2 16.2 13.2C19.845 13.2 22.8 14.2745 22.8
+      16.68V19.2Z"
+      fill="black"
```

Comment:
```suggestion
      fill="currentColor"
```

See the big comment block above